pkg/commandhandler/cmd.go

```diff
@@ -70,10 +71,13 @@ func execCmdInDirWithAction(cmd, workingDir, logDir string, waitForCompletion bo
 			stdOutFile.Close()
 		}
 
-		stdErrFile, err3 := os.OpenFile(errFileName, os.O_RDONLY, constants.FilePermissions_UserOnly_ReadWrite)
+		stdErrContents, err3 := os.ReadFile(errFileName)
 		if err3 == nil {
-			el.InfoFromStream("stderr:", stdErrFile)
-			stdErrFile.Close()
+			el.Info("stderr: %s", stdErrContents)
```

Comment:
Unfortunately, my fear here is that we've dodged a security issue with a bug that causes these events to not reach Kusto. There's a high likelihood that these events will contain security credentials.

Please check with Norberto and Simon on whether there's any plan to scrub these generically for creds. Otherwise, I fear we may have to replicate the logic we have for the BI data to scrub them here.